src/Microsoft.Framework.OptionsModel/ConfigurationBinder.cs

```diff
@@ -200,7 +201,12 @@ private static object CreateValueFromConfiguration(Type type, string value, ICon
                 }
                 else
                 {
+
+#if DNX451 || DNXCORE50 || NET45
+                    return TypeDescriptor.GetConverter(type).ConvertFromInvariantString(value);
+#else
                     return Convert.ChangeType(configuration.Get(null), type);
```

Comment:
If there are any cases in which we don't support the `TypeConverter`, as a workaround, we could special case those types. For example, for `Guid` we can call `Guid.Parse`